docs/features/search_chaining.rst

```diff
@@ -56,19 +56,41 @@ this give a speed-up in log likelihood evaluation.
   :width: 600
   :alt: Alternative text
 
+We first compose the model, which only represents the left hand `Gaussian`.

```

Comment:
What's the advantage of RST over Markdown?

Reply:
readthedocs doesn't support Markdown

Reply:
Ah right

Reply:
Ok maybe it does, anyway we're way too far gone to swap now haha

Reply:
Have you seen our fancy new readthedocs theme btw?

https://pyautofit.readthedocs.io/en/latest/